GMCPagingScrollView/GMCPagingScrollView.m

```diff
@@ -152,31 +152,31 @@ - (CGRect)frameForPageAtActualIndex:(NSUInteger)index {
 }
 
 - (void)layoutSubviews {
-	[super layoutSubviews];
-	
-	CGRect frameForScrollView = [self frameForScrollView];
-	if (!CGRectEqualToRect(self.scrollView.frame, frameForScrollView)) {
+    self.inLayoutSubviews = YES;
```

Comment:
I tested this on 6,7,8 iPads and 7,8 iPhones in both landscape and portrait and didn't see any problems. Is there anything specific I should look for to indicate its not working right?